psycopg_pool/psycopg_pool/pool_async.py

```diff
@@ -70,7 +76,8 @@ def __init__(
                     " https://www.psycopg.org/psycopg3/docs/advanced/pool.html"
                     "#pool-sqlalchemy for a workaround."
                 )
-
+        self.conninfo = conninfo
+        self.kwargs: AsyncKwargsParam | None = kwargs
```

Comment:
I don't think this type definition is necessary as it's the same in the function argument.

```suggestion
        self.kwargs = kwargs
```